db.go

```diff
@@ -91,24 +91,30 @@ func (rnr *dbRunner) Run(ctx context.Context, q *dbQuery) error {
 					case []byte:
 						s := string(v)
 						t := strings.ToUpper(types[i].DatabaseTypeName())
-						switch t {
-						case "TEXT", "CHAR", "VARCHAR", "NVARCHAR":
+						switch {
+						case strings.Contains(t, "TEXT") || strings.Contains(t, "CHAR") || t == "TIME": // MySQL8: ENUM = CHAR
 							row[c] = s
-						case "DATETIME":
+						case t == "DECIMAL" || t == "FLOAT" || t == "DOUBLE": // MySQL: NUMERIC = DECIMAL
+							num, err := strconv.ParseFloat(s, 64)
+							if err != nil {
+								return fmt.Errorf("invalid column: evaluated %s, but got %s(%v): %w", c, t, s, err)
+							}
+							row[c] = num
+						case t == "DATETIME": // TODO: Need test case
```

Comment:
@k2tzumi If possible, can you tell me the database where the following URL error occurred?

https://github.com/k1LoW/runn/pull/129#issuecomment-1234057949

Reply:
The following versions and images were verified
`public.ecr.aws/docker/library/mysql:8.0.29`

Reply:
Thank you!

Reply:
@k2tzumi  Maybe this case statement is not working in the code.

https://github.com/k1LoW/runn/pull/135#issuecomment-1236901855

Can I try to turn it off in this Pull Request?

Reply:
https://github.com/k1LoW/runn/releases/tag/v0.32.2

I verified the above tag with the following error

```
invalid column: evaluated created_at, but got DATETIME(2022-09-05 22:46:41): strconv.Atoi: parsing "2022-09-05 22:46:41": invalid syntax
```

It worked on the local environment image `mysql:8.0.29`
Maybe there is a subtle difference from the image of AWS..

> Can I try to turn it off in this Pull Request?

We have not yet incorporated this into the actual scenario, so you can turn it off.

Reply:
Oops? I tried to match the local image, but it worked on this branch too!

```
-----START ROWS-----
+-----------+-------------------------------+-------------------------------+
| is_locked |          created_at           |          updated_at           |
+-----------+-------------------------------+-------------------------------+
|         1 | 2022-09-05 23:01:16 +0000 UTC | 2022-09-05 23:01:16 +0000 UTC |
+-----------+-------------------------------+-------------------------------+
(1 row)
-----END ROWS-----
```

On CI, there may be a problem with locale combinations.

Reply:
> We have not yet incorporated this into the actual scenario, so you can turn it off.

Thank you.
I will delete the CASE statement once and wait for the problem you encountered to occur.